osu.Game/Screens/Play/ReplaySettings/CollectionSettings.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays.Music;
+using System.Collections.Generic;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.Play.ReplaySettings
+{
+    public class CollectionSettings : SettingsDropdownContainer
+    {
+        protected override string Title => @"collections";
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            Add(new OsuSpriteText

```

Comment:
With my suggestion of `Content => content` in the other review comment, you can tidy this up using
`Children = new Drawable[] { ... };`